Assets/HoloToolkit/Input/Scripts/Focus/SimpleSinglePointerSelector.cs

```diff
@@ -86,7 +87,27 @@ void ISourceStateHandler.OnSourceLost(SourceStateEventData eventData)
 
         void IInputHandler.OnInputUp(InputEventData eventData)
         {
-            // Nothing to do on input up.
+            // Since this input resulted in a pointer change, we mark the event as used to
+            // prevent it from falling through to other handlers to prevent potentially
+            // unintended input from reaching handlers that aren't being pointed at by
+            // the new pointer.
+            if (pointerWasChanged)
+            {
+                pointerWasChanged = false;
+                eventData.Use();
+            }
+        }
+
+        void IInputClickHandler.OnInputClicked(InputClickedEventData eventData)
+        {
+            // Since this input resulted in a pointer change, we mark the event as used to
+            // prevent it from falling through to other handlers to prevent potentially
+            // unintended input from reaching handlers that aren't being pointed at by
+            // the new pointer.
+            if (pointerWasChanged)
+            {
+                eventData.Use();
```

Comment:
I believe this functionality diverges from how it works in the shell (though it's been a while since I've tried it, so it has possibly changed). Do we want that?

Reply:
How so?

Do you really want a click while changing to the new pointer?

Reply:
I just tested it, and the shell transfers the pointer on input down, while providing a click/up on input up, matching how the Toolkit has it before this change.

I think we should match the experience of the shell has much as possible, so things are familiar. I'm open to discussion on that though.

Reply:
So we want to click, even though we're changing pointers?

Reply:
Should our implementations differ from what's familiar in the shell?

Reply:
Idk what the shell implementation is.  I'd have to test it out. Personally I think we should match the shell.

Reply:
The shell implementation is how I described it above:

> I just tested it, and the shell transfers the pointer on input down, while providing a click/up on input up, matching how the Toolkit has it before this change.

Reply:
Will revert